docs/integrations/sources/close-com.md

```diff
@@ -1,29 +1,51 @@
 # Close.com
 
+This page contains the setup guide and reference information for the [Close.com](https://www.close.com/) source connector.
+
 ## Prerequisites
 
-* Close.com Account
 * Close.com API Key
 
-Visit the [Close.com API Keys page](https://app.close.com/settings/api/) in the Close.com dashboard to access the secret key for your account. Secret key will be prefixed with `api_`.
-See [this guide](https://help.close.com/docs/api-keys) if you need to create a new one.
-
 We recommend creating a restricted key specifically for Airbyte access. This will allow you to control which resources Airbyte should be able to access. For ease of use, we recommend using read permissions for all resources and configuring which resource to replicate in the Airbyte UI.
 
 ## Setup guide
 
-1. [Log into your Airbyte Cloud](https://cloud.airbyte.com/workspaces) account.
-2. In the left navigation bar, click **Sources**. In the top-right corner, click **+new source**.
-3. On the Set up the source page, enter the name for the Close.com connector and select **Close.com** from the Source type dropdown.
-4. Fill in the API Key and Start date fields and click **Set up source**.
+### Step 1: Set up your Close.com API Key
+1. [Log in to your Close.com](https://www.close.com) account.
+2. At the bottom of the left navbar, select **Settings**.
+3. In the left menu, select **Developer**.
+4. At the top of the page, click **+ New API Key**.
+
+:::caution
+For security purposes, the API Key will only be displayed once upon creation. Be sure to copy and store the key in a secure location.
+:::
+
+For further reading on creating and maintaining Close.com API keys, refer to the 
+[official documentation](https://help.close.com/docs/api-keys-oauth).
+
+### Step 2: Set up the Close.com connector in Airbyte
```

Comment:
I really like this pattern. I think we should update our docs template to match this pattern of 
`step 1: get pre-requisite 1`
`step 2: get pre-requisite 2`
...
`step N: setup the connector in Airbyte`

mind making that change [here](https://hackmd.io/Bz75cgATSbm7DjrAqgl4rw?view)?